test/functional/interface_zmq.py

```diff
@@ -488,7 +517,7 @@ def test_multiple_interfaces(self):
         subscribers = self.setup_zmq_test([
             ("hashblock", "tcp://127.0.0.1:28334"),
             ("hashblock", "tcp://127.0.0.1:28335"),
-        ])
+        ], sync_blocks=False)
```

Comment:
nit, maybe add a comment clarifying why sync_blocks must be false (the test hangs without it)

Reply:
Thanks, done.